bin/testFrameworkOutput.cc

```diff
@@ -199,6 +199,9 @@ TEST_CASE("Check if trees are equals", "[diff]") {
     REQUIRE(ref_tree);
 
     TTree* test_tree = (TTree*) test_file->Get("t");
+    if ( ! test_tree ) {
```

Comment:
Is this run only as a test? If not, then it's probably better to reorder it according to the most common case (i.e. if usually the tree is framework/t, then have that as a default, otherwise we have one or two additional operations every time). If not, it does not really matter

Reply:
Yes, this is the code that checks if the trees are equal in the tests... but good point, I updated these to check "framework/t" and only if that fails "t" (if the tests pass I will update the reference trees and remove the fallback)